src/components/Button/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```

Comment:
The component should not be on the index file

Reply:
If we will export it from Button.tsx
Then while importing this component we would have to do something like
import Button from 'src/component/Button/Button'

Where as exporting it from index.tsx will reduce the last Button path from import statement

***Should we*** create both Button.tsx and Index.tsx and export Button from Index.tsx

Reply:
Yes its will be like a single folder exporting all the components and props through the index.ts file inside it. but will be much more maintainable